test/kubernetes-test.sh

```diff
@@ -32,16 +32,21 @@ helm template --validate install/kubernetes/cilium \
 
 kubectl apply -f cilium.yaml
 
-while true; do
-    result=$(kubectl -n kube-system get pods -l k8s-app=cilium | grep "Running" -c)
-    echo "Running pods ${result}"
-    if [ "${result}" == "2" ]; then
+runningPods="0"
 
-        echo "result match, continue with kubernetes"
-        break
-    fi
+pollCiliumPods () {
+  until [ "${runningPods}" == "2" ]
+  do
```

Comment:
```suggestion
  until [ "${runningPods}" == "2" ]; do
```

Reply:
I'll just fix them. What should be the default timeout?

Reply:
5 mins seems reasonable to me